taipy/core/data/csv.py

```diff
@@ -27,7 +27,7 @@
 from .data_node_id import DataNodeId, Edit
 
 
-class CSVDataNode(DataNode, _FileDataNodeMixin, _TabularDataNodeMixin):
+class CSVDataNode(_FileDataNodeMixin, DataNode, _TabularDataNodeMixin):
```

Comment:
hmm? Don't we initiated them all manually? so the order shouldn't be an issue?

Reply:
It does in this case.
So we have `_upload()` in both `DataNode` and `_FileDataNodeMixin`.
If we put DataNode first, when calling `CSVDataNode._upload()`, it will always go to `DataNode._upload()`, which is not what we want here.